copyright.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+function read_dir(){
+for file in `ls $1` #注意此处这是两个反引号，表示运行系统命令
```

Comment:
Comments in english 🙏

Reply:
This script file is not supposed to be committed, I already removed them.